adsc/adsc.go

```diff
@@ -586,6 +586,10 @@ func (a *ADSC) ack(msg *discovery.DiscoveryResponse, names []string) {
 	watch.lastNonce = msg.Nonce
 	watch.lastVersion = msg.VersionInfo
 	a.watches[msg.TypeUrl] = watch
+	// Incremental EDS can send partial endpoints, but in ack envoy should ack for all.
```

Comment:
This should be for all types really, its not EDS specific?

Reply:
Because currently only eds can be sent incrementally

Reply:
SDS can as well. But I don't think we handle it here.

Also RDS can in envoy - but we don't do it in istio